docs/user/README.md

```diff
@@ -217,16 +212,18 @@ writer.close()
 ```
 
 While this example simply ingests three data points from memory, it is simple to extend such that it reads from other
-data sources. For example, [this Python script](https://github.com/ModelarData/Utilities/blob/main/Apache-Parquet-Loader/main.py)
-makes it simple to bulk load time series from Apache Parquet files with the same schema by reading the Apache Parquet
-files, creating a model table that matches their schema if it does not exist, and transferring the data in the Apache
-Parquet files to `modelardbd` using Apache Arrow Flight.
-
-Time series can also be ingested into `modelardbd` using [Telegraf](https://www.influxdata.com/time-series-platform/telegraf/)
-with the [Apache Arrow Flight output plugin](https://github.com/ModelarData/Telegraf-Output-Apache-Arrow-Flight).
-By using Telegraf, data points can be efficiently streamed into `modelardbd` from a large
-[collection of data sources](https://www.influxdata.com/time-series-platform/telegraf/telegraf-input-plugin/)
-such as [MQTT](https://mqtt.org/) and [OPC-UA](https://opcfoundation.org/about/opc-technologies/opc-ua/).
+data sources. For example, [this Python
+script](https://github.com/ModelarData/Utilities/blob/main/Apache-Parquet-Loader/main.py) makes it simple to bulk load
+time series from Apache Parquet files with the same schema by reading the Apache Parquet files, creating a model table
+that matches their schema if it does not exist, and transferring the data in the Apache Parquet files to `modelardbd`
+using Apache Arrow Flight.
+
+Time series can also be ingested into `modelardbd` using
```

Comment:
I agree, there may be ways to improve the plugin and/or make it simpler to use. Also, since a significant amount of configuration is required to use the plugin as it is an [external plugin](https://github.com/influxdata/telegraf/blob/master/docs/EXTERNAL_PLUGINS.md), it may be worth spending some time make it good enough to be included into Telegraf and [contribute it](https://docs.influxdata.com/telegraf/v1/contribute/), make it easy to build a version of Telegraf with the plugin included, or provide configuration and/or scripts that make deployment of Telegraf with the plugin easier. It seems appropriate to create an issue to collect ideas about this and discuss them, however, I am afraid that it will be forgotten if we create it in [ModelarData/Telegraf-Output-Apache-Arrow-Flight](https://github.com/ModelarData/Telegraf-Output-Apache-Arrow-Flight). So should we create an issue in this repository and include updating the documentation as a task?

Reply:
I think the Telegraf functionality is relevant enough for this repository to create it here so that should be fine.

Reply:
Fixed by creating #248.